docker-java-api/src/main/java/com/github/dockerjava/api/model/ContainerSpec.java

```diff
@@ -161,6 +161,14 @@ public class ContainerSpec extends DockerObject implements Serializable {
     @JsonProperty("Configs")
     private List<ContainerSpecConfig> configs;
 
+    /**
+     * @since 1.37
```

Comment:
```suggestion
     * @since 1.38
```

found it at 1.38 https://docs.docker.com/engine/api/v1.38/#tag/Service/operation/ServiceCreate